Translation/Extractor/File/TwigFileExtractor.php

```
@@ -70,10 +80,8 @@ public function enterNode(\Twig_NodeInterface $node, \Twig_Environment $env)
                 $arguments = $node->getNode('arguments');
                 if ($arguments->hasNode($index)) {
                     $argument = $arguments->getNode($index);
-                    if (!$argument instanceof \Twig_Node_Expression_Constant) {
+                    if (!$this->checkNodeIsConstant($argument)) {
                         return $node;
-                        // FIXME: Throw exception if there is some way for the user to turn this off
-                        //        on a case-by-case basis, similar to @Ignore in PHP
```

Im not too sure either. 

I am not a fan of silencing exceptions. Can @gnat42 or @gouaille give some thoughts on this PR?